nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/QueryCassandra.java

```diff
@@ -132,6 +131,20 @@
 
     private final static List<PropertyDescriptor> propertyDescriptors;
 
+    // Relationships
+    public static final Relationship REL_SUCCESS = new Relationship.Builder()
```

Comment:
These relationships are available (albeit with more generic descriptions) in the AbstractCassandraProcessor class. If we need more specific documentation then we should remove the common relationships and provide specific documentation/relationships in all Cassandra processors.